packages/core/src/Site/index.js

```diff
@@ -889,6 +919,35 @@ class Site {
     });
   }
 
+  /**
+   * Handles the reloading of ignore attributes
+   */
+  async handleIgnoreReload(oldIgnore) {
+    const assetsToRemoved = _.difference(this.siteConfig.ignore, oldIgnore);
+
+    if (!_.isEqual(oldIgnore, this.siteConfig.ignore)) {
```

Comment:
Hmm, is this check neccessary? Would there be a case where `assetsToRemoved` is truthy but `oldIgnore == this.siteConfig.ignore`?

Reply:
I believe this check is necessary. For example, if one or more entries are removed from `ignore`, then `assetsToRemove` will be empty but `oldIgnore !== this.siteConfig.ignore` so the assets will be rebuilt. Another scenario is when `oldIgnore == this.siteConfig.ignore`, then in this case, if they are equal then no assets will need to be built or removed.

Reply:
Ah okay got it. We can leave this in then 👍